graphql/schema.graphql

```diff
@@ -495,6 +501,21 @@ type SiteBanner {
   theme: String!
 }
 
+type HostEvents {
+  eventLogEntries: [EventLogEntry!]!
+}
+
+type EventLogEntry {
+	id: String!
+	resourceType: String!
+	processedAt: Time!
+	timestamp: Time!
+	resourceId: String!
+	eventType: String!
+	data: Any!
```

Comment:
I think we can avoid adding the Any scalar type. checkout TaskEventLogData

Reply:
^yes to what arjun said. and if there were not an existing struct to represent data, then we would create one that describes the fields we need from `data`